DataFormats/TotemRPReco/src/classes_def.xml

```diff
@@ -0,0 +1,31 @@
+<lcgdict>
+  <class name="TotemRPCluster"/>
+  <class name="edm::DetSet<TotemRPCluster>"/>
+  <class name="std::vector<TotemRPCluster>"/>
+  <class name="std::vector<edm::DetSet<TotemRPCluster> >"/>
+  <class name="edm::DetSetVector<TotemRPCluster>"/>
+  <class name="edm::Wrapper<edm::DetSetVector<TotemRPCluster> >"/>
+
+  <class name="TotemRPRecHit"/>
+  <class name="edm::DetSet<TotemRPRecHit>"/>
+  <class name="edm::DetSetVector<TotemRPRecHit>"/>
+  <class name="std::vector<edm::DetSet<TotemRPRecHit> >"/>
+  <class name="edm::Wrapper<edm::DetSetVector<TotemRPRecHit> >"/>
+  <class name="std::vector<TotemRPRecHit>"/>
+  <class name="std::vector<const TotemRPRecHit*>"/>
+	  
+  <class name="std::pair<__gnu_cxx::__normal_iterator<const TotemRPRecHit*,std::vector<TotemRPRecHit> >,__gnu_cxx::__normal_iterator<const TotemRPRecHit*,std::vector<TotemRPRecHit> > >"/>
```

Comment:
is this dictionary really needed?
It can't be persisted. (same for ``` vector<const TotemRPRecHit*> ``` )
Which code needs it?
edm::Ref or edm::RefVector can be used instead
(or edm::Ptr if the code needs to handle polymorphic cases)